godot-macros/Cargo.toml

```diff
@@ -31,7 +32,7 @@ godot-bindings = { path = "../godot-bindings", version = "=0.1.3" } # emit_godot
 
 # Reverse dev dependencies so doctests can use `godot::` prefix.
 [dev-dependencies]
-godot = { path = "../godot", default-features = false }
+godot = { path = "../godot", default-features = false, features = ["__codegen-full"] }
```

Comment:
The CI fails without the change I made, because for some reason `cargo test` is compiling `godot-macros` with the feature, but `godot` without, so it fails to compile because `godot-macros` then generates code that's feature-gated in `godot`.

Reply:
But it worked before? So what changed that makes it necessary now? Doctests?

It's not like the macros access any library functionality from the other crates -- they just generate code.

Reply:
> But it worked before? So what changed that makes it necessary now? Doctests?
Not with my additions, no, they require enums that are only present when code-gen_full is enabled